### DISCUSSION TITLE
Support for `scrollRestoration` in turbopack.

### DISCUSSION THREAD

**Discussion by AkifumiSato** [vercel/next.js#54378] [Ideas]
### Goals

Support for `scrollRestoration` in turbopack.

### Background

Since the pages router does not publish the history key, there is no way to handle this on your own.
#47242

### Proposal

I don't think any turbopack-specific modifications are needed in terms of functionality, but I don't know.

**Comment by koichik** [comment]
If `experimental.scrollRestoration` does not affect the turbopack, so it would be nice to be added to this ignored list:
https://github.com/vercel/next.js/blob/8bbf674a4daafe7d839b55ae4670fe78d970b185/packages/next/src/lib/turbopack-warning.ts#L29-L39

**Comment by AkifumiSato** [comment]
Close.
ref #56150
